Automattic/wp-calypso
# Remove map_block_map_provider from ETK

Related to https://github.com/Automattic/jetpack/pull/30287

## Proposed Changes

This PR removes the map_block_map_provider from ETK, since it's being moved to ` jetpack-mu-wpcom`.

## Testing Instructions



- Install the ETK build from the comment below on your sandbox. 
- Verify that everything still works.
- Make sure `map_block_map_provider` is no longer available on the post-new iframe when publishing a post

<img width="593" alt="CleanShot 2023-04-26 at 09 56 21@2x" src="https://user-images.githubusercontent.com/528287/234508573-6c59a2d2-4b83-478b-9cba-fbdd428268df.png">


## Pre-merge Checklist
